python/docs/source/reference/execution.md

```diff
@@ -8,4 +8,5 @@
 
     Execution
     ResultIterator
+    record
```

Comment:
`record` is not part of execution -- it should be part of `records.md`.

Reply:
there is `record` and `Timestream.record`... 

seems like `record` should not be under the `Timestream` docs... tossed it here but maybe it should be elsewhere?

Reply:
They're both there because `record` creates a timestream and `Timestream.record` is a short-hand for doing the same thing, but computing the fields from an initial timestream. I think they both belong in `record.md` since they are both "functions for creating a recrod timestream"